### PR TITLE
ERC4907： Change status to Last Call

### PR DESCRIPTION
**Pull request by emojidao**
Change status to Last Call

### CODE REVIEW

**Review by 0xanders — APPROVED** [review]

**Review by lightclient — CHANGES_REQUESTED** [review]
In "Backward Compatibility" please replace "ERC721" with "ERC-721". Looks good otherwise.

**Review by ed738 — APPROVED** [review]

**Review by ed738 — APPROVED** [review]

**Comment by 0xanders** [comment]
Hi @lightclient  ,  I see this pr has been approved at [EIP editor apprenticeship meeting](https://github.com/ethereum-cat-herders/EIPIP/issues/135), could you pls approve this pr and set the last call date?

File: EIPS/eip-4907.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Rental NFT, ERC-721 User And Expires Extension
 description: Add a time-limited role with restricted permissions to ERC-721 tokens.
 author: Anders (@0xanders), Lance (@LanceSnow), Shrug <shrug@emojidao.org>
 discussions-to: https://ethereum-magicians.org/t/idea-erc-721-user-and-expires-extension/8572
-status: Review
+status: Last Call
```
Comment (MicahZoltu) [inline EIPS/eip-4907.md:7]:
```suggestion
status: Last Call
last-call-deadline: YYYY-MM-DD
```
This needs a last call deadline.  It should be *at least* 14 days in the future.  Due to editor response times being a bit slow for ERCs, I recommend setting it 3-4 weeks in the future so you don't run into problems.

**Review by lightclient — APPROVED** [review]

**Review by MicahZoltu — COMMENTED** [review]
````
run in Terminal：
```
truffle test ./test/test.js
```
````
the chances that this works at an undefined point in the future approach 0 as the distance between now and then approach infinity.  In other words, please remove this as it will almost certainly break, and doesn't provide significant value toward standardization.

**Comment by 0xGh** [comment]
Hi there folks! I just learnt about this proposal and I wanted to chime in and ask what would be the best place to chat about design decisions eg. `user` vs `holder` or why `useOf` doesn't return a tuple or a struct

**Comment by MicahZoltu** [comment]
> what would be the best place to chat about design decisions

The discussions-to link, which is in the header of the EIP: https://ethereum-magicians.org/t/idea-erc-721-user-and-expires-extension/8572

**Pull request merged**